src/Compilers/Core/Portable/NativePdbWriter/ComMemoryStream.cs

```diff
@@ -52,6 +63,32 @@ public void CopyTo(Stream stream)
             }
         }
 
+        public IEnumerable<ArraySegment<byte>> GetChunks()
+        {
+            int chunkIndex = 0;
+            int remainingBytes = _length;
+            while (remainingBytes > 0)
+            {
+                int bytesToCopy;
+
+                byte[] chunk;
+                if (chunkIndex < _chunks.Count)
+                {
+                    bytesToCopy = Math.Min(_chunkSize, remainingBytes);
```

Comment:
>_chunkSize [](start = 43, length = 10)

Should this be ```_chunks[chunkIndex].Length``` instead?

Reply:
All chunks are of the same length: _chunkSize.

---
In reply to: [137341916](https://github.com/dotnet/roslyn/pull/21712#discussion_r137341916) [](ancestors = 137341916)

Reply:
I guess we can use _chunks[chunkIndex].Length to make it clearer.

---
In reply to: [137376962](https://github.com/dotnet/roslyn/pull/21712#discussion_r137376962) [](ancestors = 137376962,137341916)